example-app/src/main/java/com/hcaptcha/example/MainActivity.java

```diff
@@ -98,11 +100,18 @@ public void onClickVerify(final View v) {
         }
     }
 
+    public void onTokenRelease(final View v) {
```

Comment:
`onMarkUsed` could be more appropriate here.

Reply:
Fixed